spring-integration-core/src/main/java/org/springframework/integration/expression/IntegrationEvaluationContextAwareBeanPostProcessor.java

```diff
@@ -50,4 +58,18 @@ public Object postProcessBeforeInitialization(Object bean, String beanName) thro
 	public Object postProcessAfterInitialization(Object bean, String beanName) throws BeansException {
 		return bean;
 	}
+
+	@Override
+	public void afterSingletonsInstantiated() {
```

Comment:
Yes, but pay attention to the  `IntegrationEvaluationContextAware` JavaDocs, please.